pkg/interactive/interactive.go

```diff
@@ -177,12 +177,30 @@ func GetOption(input Input) (a string, err error) {
 	if !ok {
 		dflt = ""
 	}
+	defaultMessage := ""
+	if dflt != "" {
+		defaultMessage = fmt.Sprintf("default = '%s'", dflt)
+	}
 	question := input.Question
-	if !input.Required && dflt == "" {
-		question = fmt.Sprintf("%s (optional, choose '%s' to skip selection. "+
-			"The default value will be supplied.)", question, consts.SkipSelectionOption)
+	optionalMessage := ""
+	if !input.Required {
+		optionalMessage = fmt.Sprintf("optional, choose '%s' to skip selection", consts.SkipSelectionOption)
 		input.Options = append([]string{consts.SkipSelectionOption}, input.Options...)
-		dflt = consts.SkipSelectionOption
+		if dflt == "" {
+			dflt = consts.SkipSelectionOption
+		} else {
+			optionalMessage += ". The default value will be provided"
```

Comment:
Why is the dot in the beginning of the string, shouldn't it be moved to above?

Reply:
It is below as I do not want to have a final dot if we do not have more to add :)